README.md

```diff
@@ -145,12 +145,63 @@ grunt.initConfig({
 });
 
 ```
+#### optioins.reportXML
+* Default:'path/to/file.xml'
+
+Example:
+
+```javascript
+grunt.initConfig({
+  bootlint: {
+    options: {
+      reportXML: 'report/bootlint.xml'
+    },
+    files: ['test/fixtures/**.html'],
+  },
+});
+
+```
+
+#### optioins.reportCSV
+* Default:'path/to/file.csv'
+
+Example:
+
+```javascript
+grunt.initConfig({
+  bootlint: {
+    options: {
+      reportCSV: 'report/bootlint.csv'
+    },
+    files: ['test/fixtures/**.html'],
+  },
+});
+
+```
+
+#### optioins.reportTAP
+* Default:'path/to/file.tap'
+
+Example:
+
+```javascript
+grunt.initConfig({
+  bootlint: {
+    options: {
+      reportTAP: 'report/bootlint.tap'
+    },
+    files: ['test/fixtures/**.html'],
+  },
+});
+
+```
 
 ## Contributing
 In lieu of a formal styleguide, take care to maintain the existing coding style. Add unit tests for any new or changed functionality. Lint and test your code using [Grunt](http://gruntjs.com/).
 
 ## Release History
 
+- 2015-2-3   - v0.7.0: Added downloadable report formats checkstyle format .xml, pipe delimited .csv, binary format .tap
```

Comment:
Don't need to make changes to the README here. This section is used for versioned releases only.